src/webapp/contexts/app-context.ts

```diff
@@ -2,6 +2,8 @@ import React, { useContext } from "react";
 import { CompositionRoot } from "../../CompositionRoot";
 import { User } from "../../domain/entities/User";
 import { D2Api } from "../../types/d2-api";
+// @ts-ignore
+import i18n from "../../locales";
```

Comment:
Projects based on the latest version of skeleton app should import i18n from the `i18n.ts` file.

```ts
import i18n from "../../utils/i18n";
```